tests/ui_tests/app/page/admin/locators.py

```diff
@@ -164,3 +164,68 @@ class SelectPermissionsRow:
         save_btn = Locator(
             By.XPATH, "//button[./span[contains(text(), 'Create') or contains(text(), 'Save')]]", "Save button"
         )
+
+
+class AdminPoliciesLocators:
+    """Locators for Admin Policies menu"""
+
+    create_policy_btn = Locator(By.CSS_SELECTOR, "app-add-button button", "Create policy button")
+    delete_btn = Locator(By.CSS_SELECTOR, ".controls>button", "Delete policy button")
+
+    class PolicyRow:
+        """Row with policy info"""
+
+        checkbox = Locator(By.CSS_SELECTOR, "mat-checkbox", "policy checkbox")
+        name = Locator(By.CSS_SELECTOR, "mat-cell:nth-child(2)", "policy name")
+
+    class AddPolicyPopup:
+        """Locators for creating policy popup"""
+
+        block = Locator(By.CSS_SELECTOR, "app-rbac-policy-form>mat-horizontal-stepper", "Add policy popup block")
+
+        field_error = TemplateLocator(By.XPATH, "//mat-error[contains(text(), '{}')]", 'Error "{}"')
+
+        class FirstStep:
+            """Locators for first step"""
+
+            name_input = Locator(By.CSS_SELECTOR, "input[name='name']", "Input name")
+            description_input = Locator(By.CSS_SELECTOR, "input[name='description']", "Input description")
+            role_select = Locator(By.CSS_SELECTOR, "mat-select[placeholder='Role']", "select role")
+            role_item = Locator(
+                By.XPATH, "//div[./mat-option//*[@placeholderlabel='Select role']]/mat-option", "select items for role"
+            )
+
+            users_select = Locator(By.CSS_SELECTOR, "adwp-input-select[label='User'] adwp-select", "select users")
+            users_item = Locator(By.CSS_SELECTOR, "adwp-selection-list mat-list-option", "select items for users")
+
+            group_select = Locator(By.CSS_SELECTOR, "adwp-input-select[label='Group'] adwp-select", "select group")
+            group_item = Locator(By.CSS_SELECTOR, "adwp-selection-list mat-list-option", "select items for group")
+
+            next_btn_first = Locator(
```

Comment:
I guess "next_btn" is enough here and below because it's already inside the "{#}Step" container, don't you think so?

Reply:
Thou description is good, it will be easier to understand the problem in a report